guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -17,6 +16,31 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## v2002.0.22
+
+-  {:.new}**Changes to the ece-tools composer package**
+
+   We changed the structure of the {{ site.data.var.ct }} package to reduce complexity for scheduling updates and working with community contributions.
+
+   -  {:.new}{{ site.data.var.ece }} patches are no longer included in the {{ site.data.var.ct }} ece-tools package to allow Magento to manage and deliver patches independently without requiring an {{ site.data.var.ct }} package release. These patches are now delivered in a separate Magento Cloud Patches module that is a required dependency for the {{ site.data.var.ct }} package.
+
+   -  {:.new}Updated the composer requirements for the {{ site.data.var.ct }} package to add a dependency for the Magento Cloud Patches module `magento/magento-cloud-patches`.
+
+-  {:.new}**Magento Cloud Patches package**–Initial release of the Magento Cloud Patches module which delivers Magento-provided patches and critical fixes in a separate composer package. Module source files are available in the [Magento Cloud Patches](https://github.com/magento/magento-cloud-patches) GitHub repository.
+
+   Magento Cloud Patches, version 1.0 adds the following patches for {{ site.data.var.ee }} applications hosted on the Cloud platform.
+
+   -  {:.new}<!--MAGECLOUD-4649-->**Page Builder security patches for 2.3.1.x and 2.3.2.x releases**–Fixes an issue in Page Builder preview that allows unauthenticated users to access some templating methods that can be used to trigger arbitrary code execution over the network (RCE) resulting in global information leaks. This issue can occur when using unsupported versions of Page Builder, such as Page Builder Beta with {{ site.data.var.ee }} versions 2.3.1 and v2.3.2.
```

Comment:
```suggestion
   -  {:.new}<!--MAGECLOUD-4649-->**Page Builder security patches for 2.3.1.x and 2.3.2.x releases**–Fixes an issue in Page Builder preview that allows unauthenticated users to access some templating methods that can be used to trigger arbitrary code execution over the network (RCE) resulting in global information leaks. This issue can occur when using unsupported versions of Page Builder, such as Page Builder Beta with {{ site.data.var.ee }} versions 2.3.1 and 2.3.2.
```

Reply:
I am not sure last sentence is correct. This happens not to beta versions but actually released but outdated version, used as part of 2.3.1 and 2.3.2. We cannot name those as beta though, they were officially released as GA in 2.3.1 / 2.3.2 but the fix is available in 2.3.3 and 2.3.2-p2 only, so old versions are vulnerable

Reply:
I got the reference to the beta from here: https://magento.com/security/patches/latest-magento-security-update-helps-protect-recently-reported-rce-vulnerability -- 

![image](https://user-images.githubusercontent.com/6391769/68873886-efd10b80-06c5-11ea-92a1-da682226b302.png)

Reply:
It's mentioned there separately though, not in context of "having beta module in 2.3.1 and 2.3.2". Also we don't patch beta modules, so let's just remove it, so it's not confusing